### ISSUE TITLE
Ticket query links

### ISSUE THREAD

**Issue by kwankyu** [sagemath/trac-to-github#31]
Here

https://github.com/kwankyu/trac_to_gh/wiki/TicketReports#my-tickets:~:text=%5Breport%3A75%20Tickets%20needing%20review%5D%20(This%20is%20the%20Trac%20macro%20TicketQuery%20that%20was%20inherited%20from%20the%20migration%20called%20with%20arguments%20(status%3Dneeds_review%2Cformat%3Dcount))%20%2D%2D%20most%20recent%20first

The link is 

```
[This is the Trac macro TicketQuery that was inherited from the migration](https://trac.sagemath.org/wiki/WikiMacros#TicketQuery-macro)
```

Perhaps this should be

```
[This is the Trac macro *TicketQuery* that was inherited from the migration called with arguments (status=needs_review,format=count)](https://trac.sagemath.org/wiki/WikiMacros#TicketQuery-macro) 
```

?

**Comment by kwankyu** [comment]
Fixed in PR #36

**Comment by mkoeppe** [comment]
Closed by #36
